### PR TITLE
Fix ci

### PR DESCRIPTION
**Pull request by trungleduc**
Skip releasing the npm package.

### CODE REVIEW

File: .github/workflows/publish-release.yml [hunk 1]
```diff
@@ -34,7 +34,7 @@ jobs:
           token: ${{ steps.app-token.outputs.token }}
           branch: ${{ github.event.inputs.branch }}
           release_url: ${{ github.event.inputs.release_url }}
-          steps_to_skip: ${{ github.event.inputs.steps_to_skip }}
+          steps_to_skip: '${{ github.event.inputs.steps_to_skip }},build_npm,check_npm'
```
[the remark this answers is not in the document]
Comment (jtpio) [reply .github/workflows/publish-release.yml:37]: Or use the releaser config in `pyproject.toml` to skip these steps.

[the remark this answers is not in the document]
Comment (trungleduc) [reply .github/workflows/publish-release.yml:37]: Ah, I did not know about these possibilities

**Comment by jtpio** [comment]
Maybe skipping the npm steps completely can also help make CI a bit faster (example skipping `check-python` here: https://github.com/voila-dashboards/voici/blob/df8c17b3caab83b0de6493e096278e0fd8300784/pyproject.toml#L15)

**Comment by jtpio** [comment]
Although thinking more about it, it can still be useful to perform these checks to make sure the npm package builds fine and is correctly structured.

**Pull request closed without merging**